package.json

```diff
@@ -137,8 +137,10 @@
     "node-sass": "^4.12.0",
     "npm-run-all": "^4.1.5",
     "object-hash": "^1.3.1",
-    "postcss": "^7.0.18",
+    "postcss": "^8.0.0",
```

Comment:
@BPScott do you know of any issues we might see by bumping postcss?

Reply:
You'll need to also update postcss-modules and @shopify/postcss-config too.


https://github.com/Shopify/loom/pull/307 is me updating postcss 7 to 8 along with those other two over in a loom plugin.